mkt/developers/forms.py

```diff
@@ -361,7 +362,10 @@ def save(self, addon, commit=True):
         contact = self.cleaned_data.get('mozilla_contact')
         if contact is not None:
             updates['mozilla_contact'] = contact
+        priority_before = addon.priority_review
         addon.update(**updates)
+        if self.cleaned_data.get('priority_review') and not priority_before:
```

Comment:
Can you inline `addon.priority_review` here? Looks like `priority_before` is only used once.